packages/experiments/src/components/UnifiedPicker/hooks/useSelectedItems.ts

```diff
@@ -55,9 +55,12 @@ export const useSelectedItems = <T extends {}>(
   };
 
   const removeItems = (itemsToRemove: any[]): void => {
-    const currentitems: T[] = [...items];
-    const updatedItems: T[] = currentitems.filter(item => itemsToRemove.indexOf(item) === -1);
-
+    const currentItems: T[] = [...items];
```

Comment:
If we add the comment as suggested above we should also add a comment over here indicating why we chose to do this in this way instead of how we were doing before.